script/build.go

```diff
@@ -50,7 +51,12 @@ func mainBuild() {
 	cmd, _ := exec.Command("git", "rev-parse", "--short", "HEAD").Output()
 
 	if len(cmd) > 0 {
-		LdFlag = strings.TrimSpace("-X github.com/git-lfs/git-lfs/config.GitCommit=" + string(cmd))
+		LdFlags = append(LdFlags, "-X", strings.TrimSpace(
+			"github.com/git-lfs/git-lfs/config.GitCommit="+string(cmd),
+		))
+	}
+	if !*BuildDwarf || *BuildAll {
+		LdFlags = append(LdFlags, "-s", "-w")
```

Comment:
Does it really make sense to only use `-s`, `-w` for `BuildAll`? From the outside, it's probably hard to comprehend why builds for a single platform are suddenly not stripped.

IMO the current default should be to always strip, regardless of `BuildAll`, unless `BuildDwarf` is given. And while at it, I'd probably rename `BuildDwarf` to something like `BuildDebug`.

If that turns out to be inconvenient during development, invert the logic and to not strip by default unless a `BuildRelease` is given, independently of `BuildAll` being given or not.